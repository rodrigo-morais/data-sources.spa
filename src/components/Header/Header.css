header {
  grid-column: 1;
  grid-row: 1;
  height: 12%;
  background-color: #ffffff;
  border-bottom: solid 1px #C7C5C5;
  color: #49E5CD;
  padding-left: 2%;
  font-size: 10px;
}
