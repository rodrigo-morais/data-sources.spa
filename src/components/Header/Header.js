import Vue from 'vue'
import Component from 'vue-class-component'

export default Component({
  name: 'Header',
})(class Headerr extends Vue {
  title = 'My Data Sources'
})
