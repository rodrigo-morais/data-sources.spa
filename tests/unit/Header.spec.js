import { shallowMount } from '@vue/test-utils';
import { createRenderer } from 'vue-server-renderer';
import Header from '@/components/Header/index.vue';

describe('Header.vue', () => {
  it('renders according to design', () => {
    const renderer = createRenderer();
    const wrapper = shallowMount(Header);
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err);
      expect(str).toMatchSnapshot();
    });
  });
});
