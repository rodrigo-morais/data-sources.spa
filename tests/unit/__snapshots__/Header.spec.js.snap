// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBar.vue renders according to design 1`] = `
<header>
  <h1>My Data Sources</h1>
</header>
`;
